packages/core/src/service/helpers/verification-token-generator/verification-token-generator.ts

```diff
@@ -28,7 +28,11 @@ export class VerificationTokenGenerator {
      * as specified in the VendureConfig.
      */
     verifyVerificationToken(token: string): boolean {
-        const duration = ms(this.configService.authOptions.verificationTokenDuration as string);
+        const duration =
```

Comment:
just to ease the readability slightly, could we do:

```ts
const { verificationTokenDuration } = this.configService.authOptions;
```

to avoid repeating the full object path 3 times? Same with the other one too.

Reply:
yeah definitely! I thought about doing so but wanted to keep it in one line so didn't do it then. Will update shortly.

Reply:
what do you think about this?
```ts
verifyVerificationToken(token: string): boolean {
    let { verificationTokenDuration } = this.configService.authOptions;
    if (typeof verificationTokenDuration === 'string') {
        verificationTokenDuration = ms(verificationTokenDuration);
    }

    const [generatedOn] = token.split('_');
    const dateString = Buffer.from(generatedOn, 'base64').toString();
    const date = new Date(dateString);
    const elapsed = +new Date() - +date;
    return elapsed < verificationTokenDuration;
}
```

for sessionDuration I would do 

```ts
const { sessionDuration } = this.configService.authOptions;
this.sessionDurationInMs = 
    typeof sessionDuration === 'string' ? ms(sessionDuration) : sessionDuration;
```

Reply:
Rather than re-assign the verificationTokenDuration variable, how about you create a new variable named verificationTokenDurationInMs, and assign it with a ternary like in the sessionDuration example? That would be even more explicit.